src/utils/utils.cairo

```diff
@@ -69,6 +69,16 @@ namespace Helpers {
         return res;
     }
 
+    func compute_minimum_word_size{range_check_ptr}(length : Uint256) -> (res : felt) {
```

Comment:
I created a util function compute_minimum_word_size that does the math in uint256. This mitigates the case of the (length + 31) / 32 not dividing evenly in felt arithmetic when the numerator is not a multiple of the denominator, which would cause overflow. If that helper looks could I can update the codebase to use it.